source/game.stationmap.bmx

```diff
@@ -3705,25 +3715,19 @@ Type TStationAntenna Extends TStationBase {_exposeToLua="selected"}
 		If HasFlag(TVTStationFlag.NO_RUNNING_COSTS) Then Return 0
 
 		Local result:Int = 0
+		Local difficulty:TPlayerDifficulty=GetPlayerDifficulty(String(owner))
 
 		'== ADD STATIC RUNNING COSTS ==
-		result :+ 5000 * Ceil(GetBuyPrice() / 25000.0)
-
+		result :+ Ceil(GetBuyPrice() / 5.0)
+		result :* difficulty.antennaDailyCostsMod
 		'== ADD RELATIVE MAINTENANCE COSTS ==
-		If GameRules.stationIncreaseDailyMaintenanceCosts
-			'the older a station gets, the more the running costs will be
-			'(more little repairs and so on)
-			'2% per day
-			Local maintenanceCostsPercentage:Int = GameRules.stationDailyMaintenanceCostsPercentage * GetAgeInDays()
-			'negative values deactivate the limit, positive once limit it
-			If GameRules.stationDailyMaintenanceCostsPercentageTotalMax >= 0
-				maintenanceCostsPercentage = Min(maintenanceCostsPercentage, GameRules.stationDailyMaintenanceCostsPercentageTotalMax)
-			EndIf
-
-			'1000 is "block size"
-			result = 1000*Int( (result * (1.0 + maintenanceCostsPercentage))/1000 )
-		EndIf
+		Local maintenanceCostPercentage:Float=difficulty.antennaDailyCostsIncrease
```

Comment:
Wollen wir den alten Kommentar (warum es "ansteigt") wieder mit reinnehmen?

Reply:
Kommentar wieder reingenommen. Beim Testen ist mir aufgefallen, dass zwar in der Karte die aktualisierten laufenden Kosten angezeigt werden, aber nicht in der Finanzübersicht.
Der Cache der Antennenkosten wurde nie gelöscht.